### ISSUE TITLE
undefined method + for nil with svg

### ISSUE THREAD

**Issue by matti** [teamcapybara/capybara#2051]
```ruby
visit "https://jsfiddle.net/gh/get/jquery/1.7.2/highslide-software/highcharts.com/tree/master/samples/highcharts/boost/line/"

within_frame "result" do
  svg = find "div svg"
  p svg
end

within_frame "result" do
  elems = all "div svg *"
  puts "got #{elems.size} elements"

  elems.each_with_index do |el, index|
    puts "index: #{index}"
    puts "-"*80
    puts el.inspect
  end
end
```

output:
```
#<Capybara::Node::Element tag="svg" path="/html/body/div/div/./*[((local-name(.) = 'svg') and (namespace-uri(.) = 'http://www.w3.org/2000/svg'))]">
got 82 elements
index: 0
--------------------------------------------------------------------------------
#<Capybara::Node::Element tag="defs" path="/html/body/div/div/./*[((local-name(.) = 'svg') and (namespace-uri(.) = 'http://www.w3.org/2000/svg'))]/./*[((local-name(.) = 'defs') and (namespace-uri(.) = 'http://www.w3.org/2000/svg'))]">
index: 1
--------------------------------------------------------------------------------
Traceback (most recent call last):
	12: from kapybara.rb:34:in `<main>'
	11: from kapybara.rb:34:in `load'
	10: from tests/svg_jsfiddle.rb:8:in `<top (required)>'
	 9: from /Users/mpa/.rvm/gems/ruby-2.5.1@kapybara/bundler/gems/capybara-51bed769a2b2/lib/capybara/dsl.rb:51:in `block (2 levels) in <module:DSL>'
	 8: from /Users/mpa/.rvm/gems/ruby-2.5.1@kapybara/bundler/gems/capybara-51bed769a2b2/lib/capybara/session.rb:418:in `within_frame'
	 7: from tests/svg_jsfiddle.rb:12:in `block in <top (required)>'
	 6: from tests/svg_jsfiddle.rb:12:in `each_with_index'
	 5: from /Users/mpa/.rvm/gems/ruby-2.5.1@kapybara/bundler/gems/capybara-51bed769a2b2/lib/capybara/result.rb:40:in `each'
	 4: from /Users/mpa/.rvm/gems/ruby-2.5.1@kapybara/bundler/gems/capybara-51bed769a2b2/lib/capybara/result.rb:40:in `each'
	 3: from tests/svg_jsfiddle.rb:15:in `block (2 levels) in <top (required)>'
	 2: from tests/svg_jsfiddle.rb:15:in `p'
	 1: from /Users/mpa/.rvm/gems/ruby-2.5.1@kapybara/bundler/gems/capybara-51bed769a2b2/lib/capybara/node/element.rb:414:in `inspect'
/Users/mpa/.rvm/gems/ruby-2.5.1@kapybara/bundler/gems/capybara-51bed769a2b2/lib/capybara/selenium/node.rb:171:in `path': undefined method `+' for nil:NilClass (NoMethodError)
```

repro in https://github.com/matti/kapybara - `ruby kapybara.rb tests/svg_jsfiddle.rb`
Sorry didn't have time to setup capybara specs (nokogiri incompatible library version...)

That is using capybara from master https://github.com/matti/kapybara/blob/master/Gemfile#L3

another repro with `ruby kapybara.rb tests/svg_local.rb` (uses the #2050 svg document)

```ruby
visit "http://localhost:4567/svg.html"

elems = all "svg *"

elems.each_with_index do |el, i|
  puts i
  puts el.inspect
end
```

```
0
Traceback (most recent call last):
	8: from kapybara.rb:34:in `<main>'
	7: from kapybara.rb:34:in `load'
	6: from tests/svg_local.rb:7:in `<top (required)>'
	5: from tests/svg_local.rb:7:in `each_with_index'
	4: from /Users/mpa/.rvm/gems/ruby-2.5.1@kapybara/bundler/gems/capybara-51bed769a2b2/lib/capybara/result.rb:40:in `each'
	3: from /Users/mpa/.rvm/gems/ruby-2.5.1@kapybara/bundler/gems/capybara-51bed769a2b2/lib/capybara/result.rb:40:in `each'
	2: from tests/svg_local.rb:9:in `block in <top (required)>'
	1: from /Users/mpa/.rvm/gems/ruby-2.5.1@kapybara/bundler/gems/capybara-51bed769a2b2/lib/capybara/node/element.rb:414:in `inspect'
/Users/mpa/.rvm/gems/ruby-2.5.1@kapybara/bundler/gems/capybara-51bed769a2b2/lib/capybara/selenium/node.rb:171:in `path': undefined method `+' for nil:NilClass (NoMethodError)
```

**Comment by twalpole** [comment]
This is because of the mixed case element name `clipPath` - hmmm.

**Comment by twalpole** [comment]
Interestingly it doesn't affect geckodriver/FF

**Comment by twalpole** [comment]
Closed via PR #2052
